### ISSUE TITLE
ASM.js content (emscripten) can't be loaded because it relies on XMLHttpRequest

### ISSUE THREAD

**Issue by arielm** [hicetnunc2000/hicetnunc#272]
Hi, I'm trying to mint a very simple ASM.js test. The index.html is [here](http://chronotext.org/lab/nft/Basic/index.html) for reference.

The problem is that it fails in Chrome with the following error:
> Access to XMLHttpRequest at 'https://cloudflare-ipfs.com/ipfs/Qmdtp3rtJhNg6BoJ6ojYetkePhvKT2kjH9eWRyb5HKZDPj/Basic.html.mem' from origin 'null' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

I had a similar problem on Amazon S3 in the past and it has been solved by allowing CORS. Can it be done on hicetnunc.xyz?

**Comment by andrevenancio** [comment]
Yeah, I think @veqtor might be the best person to reply to this, but I think this might be related with the same issue we're seeing on Safari. Anything we can do, or close it?

**Comment by veqtor** [comment]
Hmm, how did you mint this, as a html zip?

**Comment by arielm** [comment]
Yes, I minted this as a html zip: [here it is](https://drive.google.com/file/d/1P1z6nhiP0v7k5hRxWBZL53oHGjRa22Vs/view?usp=sharing)

**Comment by arielm** [comment]
Update: I tried again to mint the same [html zip](https://drive.google.com/file/d/1P1z6nhiP0v7k5hRxWBZL53oHGjRa22Vs/view?usp=sharing), now that it's possible to preview your HTML NFT before minting.

The previous "blocked by CORS policy" error is not showing again. But there is a new error:
> Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "default-src 'self' 'unsafe-inline'".

**Comment by pichiste** [comment]
CORS issue is fixed now, can you check again? thanks

**Comment by pichiste** [comment]
@andrevenancio this one can be closed, thanks!

**Comment by arielm** [comment]
I checked and the CORS issue is gone. Thanks!
But there is this new problem I described earlier (will open a new issue)
